app/javascript/packs/Profile.jsx

```diff
@@ -53,23 +64,26 @@ export default class Profile extends React.Component {
 
     e.target.innerText = 'Movie Removed';
     e.target.disabled = true;
-    setTimeout(function() { parentDiv.remove() }.bind(this), 3000);
```

Comment:
hm, just noticed this. not related to your graphql refactor, but this doesn't seem react kosher?